experiments/mcp/src/pkg/promotion.py

```diff
@@ -0,0 +1,336 @@
+# Copyright 2025 Google LLC
```

Comment:
We should probably also make these available under dev/tools for people that prefer to run them manually.  But not a blocker.

We could also consider exposing all the `dev/tools` via MCP or similar!